python/py-pkgconfig/Portfile

```diff
@@ -19,18 +20,23 @@ long_description        ${description} It can be used to check if a package\
                         requirements, query CFLAGS and LDFLAGS, and parse the\
                         output to build extensions with setup.py.
 
-checksums \
-    rmd160  3e0396838a04166e50be9c01c88a5b963dbf0cd5 \
-    sha256  03dd0cd84255b1b00b316b3db37e9b129a5621fc9c0d931b942678997fc73f93
+checksums               rmd160  6cb57d8bd0a1797781eb1d72d57ef8a88e1ad791 \
+                        sha256  5a15786f0295dc6e98b1d0a485e795fc4248f5268563e1f53a9bd07239653cbe \
+                        size    5351
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-nose
+    depends_build-append    port:py${python.version}-setuptools
     depends_run-append      port:pkgconfig
 
+    depends_test-append     port:py${python.version}-nose
+    test.run    yes
+    test.cmd    nosetests-${python.branch}
+    test.target
+
     post-destroot {
         set DOCDIR ${destroot}${prefix}/share/doc/${subport}
         xinstall -d ${DOCDIR}
         file copy ${worksrcpath}/LICENSE ${DOCDIR}
     }
+    livecheck.type    none
```

Comment:
If this is on github, is it really the case that the github livecheck will not work?